figures/scripts/fig2-mutational-landscape.R

```diff
@@ -0,0 +1,165 @@
+# Mutational Landscape Figure
+#
+# 2020
+# C. Savonen for ALSF - CCDL
+#
+# Purpose  Run steps needed to create mutational-landscape Figure.
+#
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
+
+# Establish base dir
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# Declare output directory
+output_dir <- file.path(root_dir, "figures", "pngs")
+
+######## Declare relative file paths of modules used for this figure ###########
+snv_callers_dir <- file.path(
+  root_dir,
+  "analyses",
+  "snv-callers"
+)
+mut_sig_dir <- file.path(
+  root_dir,
+  "analyses",
+  "mutational-signatures"
+)
+tmb_dir <- file.path(
+  root_dir,
+  "analyses",
+  "tmb-compare-tcga"
+)
+
+# Import specialized functions from mutational-signatures
+source(file.path(mut_sig_dir, "util", "mut_sig_functions.R"))
+source(file.path(tmb_dir, "util", "cdf-plot-function.R"))
+
+###################### Read in associated results ##############################
+# Read in PBTA TMB results
+tmb_pbta <- data.table::fread(file.path(
+  snv_callers_dir,
+  "results",
+  "consensus",
+  "pbta-snv-mutation-tmb-coding.tsv"
+)) %>%
+  # Remove Panel samples
+  dplyr::filter(experimental_strategy != "Panel")
+
+# Read in TCGA TMB results
+tmb_tcga <- data.table::fread(file.path(
+  snv_callers_dir,
+  "results",
+  "consensus",
+  "tcga-snv-mutation-tmb-coding.tsv"
+))
```

Comment:
Looks like we're using the files that are in individual analysis modules, rather than their counterparts that are included in the data download. This is exactly what we want.